portfolio/src/main/webapp/script.js

```diff
@@ -42,18 +49,24 @@ try {
 
 // Hides dropdown menu and shows main view elements.
 const hideMenuShowMain = () => {
-  dropdownMenu.style.display = 'none';
+  ensureHasClass(dropdownMenu, 'show');
```

Comment:
We'll simply crash if the class is not found?

What you can do is that ensureHasClass returns a boolean value, and then
```
if (ensureHasClass(dropdownMenu,  'show')) {
    ...
} 
```

Reply:
Haha. The page will not continue loading. Is this not acceptable? Honestly, I think this is more of an assertion than an exception. However, I am not sure how assertions work in JS. :/

What are your thoughts?

Reply:
In that case, you can use `console.assert(ensureHasClass(dropdownMenu, 'show'),...)` (not ideal, as we may not want to expose the msg to user, but currently we don't have server side logging implementation yet), which will write an error msg to your console so that at least we know what leads to a crash, and also more obvious that this is a assertion that must always pass

Reply:
Oh cool! Yes, I have read about console.assert but I couldn't find out how would that help. 😅 Thanks for the information!

Reply:
I'm not seeing changes being made. Please only reply if you've made the change :)